testcases/IplParams.py

```diff
@@ -112,7 +112,8 @@ def test_feature_enable(self):
         params = self.get_params_table()
         log.debug("List of features which are expected to be in enabled state\n{}".format(params))
         if not params:
-            self.assertTrue(False, "fw-feature set table not found")
+            # skip the test if the processor is not GA level (for such cases as op910 supports only dd2.1)
+            raise unittest.SkipTest("fw-feature set table not found, check if the processor is supported")
```

Comment:
I'd prefer it if we looked at the chip revision itself before skipping.

Reply:
> looked at the chip revision

This earlier (get_params_table) looks at POWER9, then if not revision 2.2 it returns emtpy and then determines to skip or not.

Reply:
@stewart-ibm are you thinking here about checking if the chip revision is GA level, but the feature table is wrong?